test/test_browser.py

```diff
@@ -5637,3 +5637,26 @@ def test_emrun(self):
       self.assertContained('Testing ASCII characters: !"$%&\'()*+,-./:;<=>?@[\\]^_`{|}~', stdout)
       self.assertContained('Testing char sequences: %20%21 &auml;', stdout)
       self.assertContained('hello, error stream!', stderr)
+
+  def test_worker_error_postmessage(self):
+    create_file('src.cpp', r'''
+      #include <thread>
+
+      void crash() {
+        int (*fun)(void) = nullptr;
+        (fun)();
+      }
+
+      int main() {
+        std::thread worker(crash);
+        worker.join();
+        return 0;
+      }
+      ''')
+    create_file('pre.js', r'''
+      console.log = function(message) {
+        reportResultToServer(message);
+      };
+      ''')
+    self.compile_btest(['src.cpp', '-g', '-pthread', '-sPROXY_TO_PTHREAD', '--pre-js', 'pre.js', '-o', 'src.html'])
+    self.run_browser('src.html', '/report_result?at crash()')
```

Comment:
I think these two lines can be combined into a single `self.btest(...)`

Reply:
Done. But this test doesn't work yet now